src/manipulation.js

```diff
@@ -17,7 +17,7 @@ define([
 
 var
 	rxhtmlTag = /<(?!area|br|col|embed|hr|img|input|link|meta|param)(([\w:]+)[^>]*)\/>/gi,
-	rtagName = /<([\w:]+)/,
+	rtagName = /<([\w:-]+)/,
```

Comment:
> instead of the namespace-ignorant document.createElement:

It doesn't matter if it's ignorant or not, there is no real difference between tags created with explict namespace or not, for example in `attrs` module we always use `setAttribute` which is not namespace aware, comparing to `setAttributeNS` method, which proves that this is not real use-case.

Bottom line is - `createElement` method could handle colon case and can create required element.

> so the former correctly identifies the first open tag and the latter doesn't mangle input:

I wasn't talking about `rxhtmlTag` only about `rtagName` which doesn't requires colon, because, i as i said before, we only need to indentify html tags, and ":" sign could probably be dropped from `rxhtmlTag` too, since even it wouldn't correctly do the replace operation, browser parser is still smart enough, but this requires further investigation if we would want to go down that road.

That said, i can admit, that if we will leave everything as it is now, it would be less hacky. But what we should definitely do is add tests for these cases.